packages/components/inputs/selectable-search-input/src/selectable-search-input.tsx

```diff
@@ -520,6 +543,23 @@ const SelectableSearchInput = (props: TSelectableSearchInputProps) => {
               isDisabled={props.isDisabled}
             />
           )}
+
+          {props.rightActionIcon && props.rightActionProps && (
+            <div
+              css={css`
+                order: 4;
+                margin-left: ${designTokens.spacing30};
+              `}
+            >
+              <IconButton
+                theme="primary"
```

Comment:
@FilPob do you think this should be static?
I know we're now thinking on the filters, but from the component point of view, I'm not sure if other use cases would require a different theme.
Perhaps this can also be a property which will use the "primary" value by default.

What do you think?

Reply:
Yes that was the idea right? @misama-ct . To have the `primary` theme as default, but still changeable by the user

Reply:
@FilPob Yes, it can be overridden via the `rightActionProps` properties, it accepts all configs that the IconButton does and would look like this:

```
rightActionProps={{ label: 'Click me', onClick: alert('clicked'), theme: 'whatever' }}
```

But I think @CarlosCortizasCT meant - correct me if I'm wrong Carlos - making even the component configurable, e.g `<SecondaryIconButton/>` instead of `<IconButton/>`.

Reply:
No. Sorry, that was my mistake.
I was confused because how you use the `theme` property with the `primary` value in the `IconButton` and didn't realize you're using it as the default value that can be overwritten by the `{...props.rightActionProps}` line below.

A little confusing to me (in terms of readability) but if others engineers find it easy to reason about, it's ok for me.